.eslintignore

```diff
@@ -1,26 +1,26 @@
 # unconventional js
-packages/*/blueprints/*/*files/
-vendor/
+**/blueprints/*/*files/
+**/vendor/
 
 # compiled output
+**/dist/
+**/tmp/
 /packages/-ember-data/docs/
-dist
-tmp
 
 # dependencies
-bower_components
-node_modules
+**/bower_components/
+**/node_modules/
 
 # misc
-/coverage/
+**/coverage/
```

Comment:
I'm not sure where the coverage report is generated since I don't think we've done one since moving to packages. It may still be root